mbedtls-platform-support/src/psa_external_rng.rs

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) Fortanix, Inc.
+ *
+ * Licensed under the GNU General Public License, version 2 <LICENSE-GPL or
+ * https://www.gnu.org/licenses/gpl-2.0.html> or the Apache License, Version
+ * 2.0 <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0>, at your
+ * option. This file may not be copied, modified, or distributed except
+ * according to those terms. */
+
+use mbedtls_sys::types::raw_types::{c_void, c_uchar};
+use mbedtls_sys::types::{size_t, int32_t};
+use once_cell::sync::OnceCell;
+static PSA_CALLBACK_INIT: OnceCell<()> = OnceCell::new();
+
+struct PsaExternlRng {}
```

Comment:
What's the purpose of this type? it doesn't seem to be used

Reply:
I see what you mean, I could just use a normal function for this.
will remove this.